Rubberduck.VBEEditor/SafeComWrappers/SafeComWrapper.cs

```diff
@@ -22,7 +22,7 @@ protected SafeComWrapper(T target, bool rewrapping = false)
             Target = target;
             _rewrapping = rewrapping;
 
-            if (!rewrapping && target != null)
+            if (!rewrapping && target != null && (this is IVBProject))
```

Comment:
Considering this only happens in Excel but not in Access, I wonder if we need to further restrict this to Excel only to avoid leaks in other host applications?

Reply:
I do not know of any way to figure out the host application at that point.

Reply:
Moreover, are you sure the issue does not exist in any other host but Excel?

Reply:
At this point I do know it does not occur in Access. I can test Word and PowerPoint later to ascribe. In case it matters, this was observed on Excel 2010.